pom.xml

```diff
@@ -64,15 +64,19 @@
       <artifactId>mybatis</artifactId>
       <version>3.5.7</version>
     </dependency>
-      <dependency>
-        <groupId>org.apache.logging.log4j</groupId>
-        <artifactId>log4j-api</artifactId>
-        <version>2.20.0</version>
-      </dependency>
+
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
```

Comment:
Why are we changing the log4j dependency here? I thought Andrey fixed that. Also, if it continues to be a problem we can use a multiplatform docker image for it